src/main/webapp/index.html

```diff
@@ -3,14 +3,83 @@
   <head>
     <meta charset="UTF-8">
     <title>Friends of Friends</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+  </head>
     <script src="facebook-login.js"></script>
   </head>
   <body>
-    <div class="fb-login-button" data-size="large" data-button-type="continue_with"
-        data-layout="default" data-auto-logout-link="true" data-use-continue-as="false"
-        data-scope="public_profile,email,user_friends"></div>
+    <div class="container-fluid">
+      <div class="content">
+        <nav class="fixed-style navbar navbar-expand-sm justify-content-centerS navbar-background-home">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link nav-text-style" href="#about">About</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link nav-text-style" href="#meetus">Meet the Creators</a>
+            </li>
+            <li class="nav-item">
+               <div id="login" class="fb-login-button" data-size="large" data-button-type="continue_with"
+                  data-layout="default" data-auto-logout-link="true" data-use-continue-as="false"
+                  data-scope="public_profile,email,user_friends">
+               </div>
+            </li>
+          </ul>
+        </nav>
+        <div class="home-page-content-header">
+          <h1>Friends of Friends</h1>
+          <img src="images/logo.jpg" alt="logo" class="rounded-circle image-fluid" height="100" width="120">
+        </div>
+        <img src="images/friends-of-friends.png" class="image-fluid" alt="home page image">
+        <div class="home-page-content" id="about">
+          <h2>About</h2>
+          <p>
+            Friends of Friends is a friend finding app! Simply log in with Facebook, and we will look at your friends lists and the friends lists of other users on the app 
```

Comment:
nit: if it doesn't mess up the formatting, line wrapping these lines a bit more probably would be better